### PR TITLE
chore: Add placeholder and instructions to ff payloads

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- Adds more context as to how to use feature flag payloads. The placeholders are within field now too.

## Changes
Screen Shot 2023-02-07 at 2 13 54 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by EDsCODE** [comment]
Main issue left is that there's discrepancy between multivariate and boolean flag UI. Unsure how to show the snippet and instructions for both without it getting cluttered

File: frontend/src/scenes/feature-flags/JSONEditorInput.tsx [hunk 1]
```diff
@@ -76,6 +82,11 @@ export function JSONEditorInput({
                     onChange?.(val)
                 }}
             />
+            {!focused && !value?.toString() && placeholder && (
```
Comment (liyiy) [inline frontend/src/scenes/feature-flags/JSONEditorInput.tsx:85]: What was the problem with the placeholder again?

**Comment by liyiy** [comment]
> Main issue left is that there's discrepancy between multivariate and boolean flag UI. Unsure how to show the snippet and instructions for both without it getting cluttered

Maybe like switching to multivariate snippet instructions when the user switches the flag to multivariate mode? That's what I was planning to do with the current `isFeatureEnabled` snippet instructions at least

File: frontend/src/scenes/feature-flags/JSONEditorInput.tsx (hunk 1 above)
Comment (EDsCODE) [reply frontend/src/scenes/feature-flags/JSONEditorInput.tsx:85]: There is none on the monaco editor 👻

**Comment by EDsCODE** [comment]
Switching makes sense. With JSON payloads, the problem is that boolean flag payloads have their own space and then multivariate flag payloads are specified inline so the instructions could end up in different spots

**Pull request merged**